src/main/java/bio/terra/workspace/service/workspace/flight/EnableServicesStep.java

```diff
@@ -0,0 +1,85 @@
+package bio.terra.workspace.service.workspace.flight;
+
+import static bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys.GOOGLE_PROJECT_ID;
+
+import bio.terra.cloudres.google.api.services.common.OperationCow;
+import bio.terra.cloudres.google.api.services.common.OperationUtils;
+import bio.terra.cloudres.google.serviceusage.ServiceUsageCow;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.exception.RetryException;
+import bio.terra.workspace.service.workspace.exceptions.RetryableCrlException;
+import com.google.api.services.serviceusage.v1.model.BatchEnableServicesRequest;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.time.Duration;
+
+// Enable all required services for the newly created project.
+// NOTE: This class was extracted from the CreateProjectStep class. That class will be removed with
+// PF-156, so did not refactor that class to use this step.
+public class EnableServicesStep implements Step {
+  @VisibleForTesting
+  static final ImmutableList<String> ENABLED_SERVICES =
+      ImmutableList.of("storage-api.googleapis.com");
```

Comment:
I don't think we should have this step. The buffer service should enable the relevant services for its projects. Enabling services is the operation that can take a long time, so we should do it as a part of buffering.
e.g.
https://github.com/DataBiosphere/terra-resource-buffer/blob/409a9d3147d082da6a64a348ae45bb7ce815c780/src/main/resources/config/dev/resource-config/workspace_manager_v3.yml#L26

Reply:
Ah. Somehow did not connect that. Verified storage is enabled even without this step. Deleting.